app/src/main/java/app/foodfinderapp/ui/main/ProfileFragment.kt

```diff
@@ -19,9 +19,10 @@ private const val ARG_PARAM1 = "param1"
 private const val ARG_PARAM2 = "param2"
 
 /**
- * A simple [Fragment] subclass.
- * Use the [ProfileFragment.newInstance] factory method to
- * create an instance of this fragment.
+ * A [Fragment] subclass representing the profile screen in the application.
```

Comment:
Added JavaDoc comments to MainActivity class to improve readability and maintainability.

[Source](https://kotlinlang.org/docs/coding-conventions.html#documentation-comments)